contents/docs/cdp/build/tutorial.md

```diff
@@ -4,6 +4,8 @@ sidebar: Docs
 showTitle: true
 ---
 
+> We are temporarily not reviewing new apps while we build a new export system. We'll be accepting new app contributions again soon!
```

Comment:
we will not 😅  the plan is to kill all the apps and implement as 
process_event: native functionality or some different way to do it
onEvent: via webhooks
exportEvents: via new export system
runEveryX: should be a cron somewhere outside of PostHog

```suggestion
> We are temporarily not reviewing new apps while we build a new export system.
```